apps/estuary/config/config.exs

```diff
@@ -8,4 +8,15 @@ config :estuary,
   table_writer: Pipeline.Writer.TableWriter,
   connection: :estuary_elsa
 
+# Configures the endpoint
+config :estuary, EstuaryWeb.Endpoint,
+  url: [host: "localhost"],
+  # it should be overwriten as part of deploying the platform.
```

Comment:
I don't know what secret_key_base is supposed to be, but it doesn't sound like it should be in our src code and on github.